### PR TITLE
Add lava templete of preempt-rt

### PR DESCRIPTION
**Pull request by musamaanjum**
Please find the description in individual commits.

### CODE REVIEW

File: config/runtime/tests/preempt-rt.jinja2 [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
       name: preempt-rt-prereq
-      path: inline/preempt-rt-prereq.yaml
+      path: inline/{{ node.name }}.yaml
 
     - repository: https://github.com/kernelci/test-definitions.git
       from: git
       revision: kernelci.org
-      path: automated/linux/{{ tst_group|default(tst_cmd) }}/{{ tst_cmd }}.yaml
-      name: {{ plan }}
+      path: automated/linux/{{ tst_group|default(tst_cmd) }}/{{ node.name }}.yaml
```
Comment (pawiecz) [inline config/runtime/tests/preempt-rt.jinja2:22]: Please correct me if I'm wrong - shouldn't this stay `tst_cmd` (from what I can see in https://github.com/kernelci/test-definitions/tree/master/automated/linux and https://github.com/kernelci/kernelci-pipeline/pull/648)?

**Review by pawiecz — CHANGES_REQUESTED** [review]

File: config/runtime/tests/preempt-rt.jinja2 (hunk 1 above)
Comment (musamaanjum) [reply config/runtime/tests/preempt-rt.jinja2:22]: Excellent catch. Fixing.

File: config/runtime/tests/preempt-rt.jinja2 [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: preempt-rt-prereq
+          description: Pre-requisites for PREEMPT_RT
+        run:
+          steps:
+          - apt-get update && apt-get install -y procps rt-tests
+      name: preempt-rt-prereq
+      path: inline/{{ node.name }}.yaml
```
Comment (pawiecz) [inline config/runtime/tests/preempt-rt.jinja2:17]: nit: you could safely leave the `path` from imported definition fragment ("inline/preempt-rt-prereq.yaml") - for `inline` definitions it's there only to comply with YAML structure. Let's leave this as it is now

**Review by pawiecz — APPROVED** [review]
LGTM

File: config/runtime/tests/preempt-rt.jinja2 (hunk 2 above)
Comment (musamaanjum) [reply config/runtime/tests/preempt-rt.jinja2:17]: Sure.

**Pull request merged**